libraries/bxdf/standard_surface.mtlx

```diff
@@ -4,49 +4,91 @@
-      <input name="thin_film_IOR" type="float" value="1.5" uiname="Thin Film Index of Refraction" uifolder="Thin Film" uiadvanced="true"/>
-      <input name="emission" type="float" value="0" uiname="Emission" uifolder="Emission"/>
-      <input name="emission_color" type="color3" value="1, 1, 1" uiname="Emission Color" uifolder="Emission"/>
-      <input name="opacity" type="color3" value="1, 1, 1" uiname="Opacity" uifolder="Geometry"/>
+            doc="Autodesk standard surface shader">
+      <input name="base" type="float" value="0.8" uimin="0.0" uimax="1.0" uiname="Base" uifolder="Base"
+            doc="Multiplier on the intensity of the diffuse reflection." />
+      <input name="base_color" type="color3" value="1, 1, 1" uimin="0,0,0" uimax="1,1,1" uiname="Base Color"  uifolder="Base"
+            doc="Color of the diffuse reflection." />
+      <input name="diffuse_roughness" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Diffuse Roughness"  uifolder="Diffuse" uiadvanced="true"
+            doc="Roughness of the diffuse reflection. Higher values cause the surface to appear flatter and darker." />
+      <input name="specular" type="float" value="1" uimin="0.0" uimax="1.0" uiname="Specular" uifolder="Specular"
+            doc="Multiplier on the intensity of the specular reflection." />
+      <input name="specular_color" type="color3" value="1, 1, 1" uimin="0,0,0" uimax="1,1,1" uiname="Specular Color" uifolder="Specular"
+            doc="Color tint on the specular reflection." />
+      <input name="specular_roughness" type="float" value="0.2" uimin="0.0" uimax="1.0" uiname="Specular Roughness" uifolder="Specular"
+            doc="The roughness of the specular reflection. Lower numbers produce sharper reflections, higher numbers produce blurrier reflections." />
+      <input name="specular_IOR" type="float" value="1.5" uimin="0.0" uimax="3.0" uiname="Index of Refraction" uifolder="Specular"
+            doc="Index of refraction for specular reflection." />
+      <input name="specular_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Anisotropy" uifolder="Specular" uiadvanced="true"
+            doc="The directional bias of reflected and transmited light resulting in materials appearing rougher or glossier in certain directions." />
+      <input name="specular_rotation" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
+            doc="Rotation of the axis of specular anisotropy around the surface normal." />
+      <input name="metalness" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Metalness" uifolder="Specular"
+            doc="Specifies how metallic the material appears. At its maximum, the surface behaves like a metal, using fully specular reflection and complex fresnel." />
+      <input name="transmission" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Transmission" uifolder="Transmission"
```

Comment:
Transmission is missing the uiadvanced="true" (my fault for missing this when I updated last time)